--
-- PostgreSQL database dump
--

SET statement_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

--
-- Name: SCHEMA "public"; Type: COMMENT; Schema: -; Owner: ielvnfqezwcorg
--

COMMENT ON SCHEMA "public" IS 'standard public schema';


--
-- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: 
--

CREATE EXTENSION IF NOT EXISTS "plpgsql" WITH SCHEMA "pg_catalog";


--
-- Name: EXTENSION "plpgsql"; Type: COMMENT; Schema: -; Owner: 
--

COMMENT ON EXTENSION "plpgsql" IS 'PL/pgSQL procedural language';


SET search_path = public, pg_catalog;

SET default_tablespace = '';

SET default_with_oids = false;

--
-- Name: schema_migrations; Type: TABLE; Schema: public; Owner: ielvnfqezwcorg; Tablespace: 
--

CREATE TABLE "schema_migrations" (
    "version" character varying NOT NULL
);


ALTER TABLE public.schema_migrations OWNER TO ielvnfqezwcorg;

--
-- Name: users; Type: TABLE; Schema: public; Owner: ielvnfqezwcorg; Tablespace: 
--

CREATE TABLE "users" (
    "id" integer NOT NULL,
    "first_name" character varying,
    "last_name" character varying,
    "email" character varying,
    "organization_name" character varying,
    "department" character varying,
    "position" character varying,
    "area_of_interest" "text",
    "presentation" character varying,
    "presentation_name" character varying,
    "abstract" "text",
    "visa_sponsorship" character varying,
    "hotel" character varying,
    "created_at" timestamp without time zone NOT NULL,
    "updated_at" timestamp without time zone NOT NULL,
    "password_digest" character varying,
    "remember_digest" character varying,
    "admin" boolean DEFAULT false,
    "activation_digest" character varying,
    "activated" boolean DEFAULT false,
    "activated_at" timestamp without time zone,
    "reset_digest" character varying,
    "reset_sent_at" timestamp without time zone
);


ALTER TABLE public.users OWNER TO ielvnfqezwcorg;

--
-- Name: users_id_seq; Type: SEQUENCE; Schema: public; Owner: ielvnfqezwcorg
--

CREATE SEQUENCE "users_id_seq"
    START WITH 1
    INCREMENT BY 1
    NO MINVALUE
    NO MAXVALUE
    CACHE 1;


ALTER TABLE public.users_id_seq OWNER TO ielvnfqezwcorg;

--
-- Name: users_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: ielvnfqezwcorg
--

ALTER SEQUENCE "users_id_seq" OWNED BY "users"."id";


--
-- Name: id; Type: DEFAULT; Schema: public; Owner: ielvnfqezwcorg
--

ALTER TABLE ONLY "users" ALTER COLUMN "id" SET DEFAULT "nextval"('"users_id_seq"'::"regclass");


--
-- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: ielvnfqezwcorg
--

COPY "schema_migrations" ("version") FROM stdin;
20160204163756
20160204164855
20160204165202
20160204204333
20160207093759
20160207105459
20160207113529
\.


--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: ielvnfqezwcorg
--

COPY "users" ("id", "first_name", "last_name", "email", "organization_name", "department", "position", "area_of_interest", "presentation", "presentation_name", "abstract", "visa_sponsorship", "hotel", "created_at", "updated_at", "password_digest", "remember_digest", "admin", "activation_digest", "activated", "activated_at", "reset_digest", "reset_sent_at") FROM stdin;
4	Johannes	Soeding	soeding@mpibpc.mpg.de	Max-Planck institute for biophysical chemistry	Quantitative and computational biology group	Group leader	Transcriptional regulation\r\nProtein bioinformatics\r\nSystems medicine	oral	Bayesian higher-order models consistently outperform PWMs at predicting regulatory motifs in nucleotide sequences	Position weight matrices (PWMs) are the standard model for DNA and RNA regulatory motifs. In PWMs nucleotide probabilities are independent of nucleotides at other positions. Models that account for dependencies need many parameters and are prone to overfitting. A Bayesian treatment of motif learning leads naturally to inhomogeneous interpolated Markov models (iIMMs) that use order-(k-􀀀1) probabilities as priors for order k. This allows iIMMs to automatically adapt model complexity to the amount of available data. We derive an iIMM-based EM algorithm for de-novo discovery of enriched motifs. For transcription factor binding, iIMMs achieve significantly (p<0.063) higher cross-validated partial AUC than PWMs in 97% of 446 ChIP-seq ENCODE datasets and improve performance by 36% on average. iIMMs also learn complex multipartite motifs, improving predictions of transcription start sites, polyadenylation sites, bacterial pause sites, and RNA binding sites by 26% -􀀀 101%. These results argue in favour of generally replacing PWMs by iIMMs.	yes	yes	2016-02-16 09:13:08.681994	2016-02-16 09:14:41.567942	$2a$10$2h3ZkGUtKu18o2Jz.HanWOF35bwsJOSxdnvaVm70n3IQf/02Ol4si	\N	f	$2a$10$q6rBEi5PxhCXc3j5Uc0Av.Ja4XlhiSyAATKHlFfeOR.8Ou.jMo3ZG	t	2016-02-16 09:14:41.566846	\N	\N
1	Alexey	Sokolov	sokolovbiotech@gmail.com	FBRAS	lab of epigenetics	junior researcher	epigenetics, machine learning	poster	New pipeline for efficient splicing events calling	test	no	no	2016-02-08 09:50:50.823659	2016-03-01 09:47:46.447381	$2a$10$s5IJUNI/MA2284Pr3nAlveZtEymC.AKiYW1soDATrY2cms6HRSnJ2	\N	t	$2a$10$FE917Z.nx/2MTrYxSNSKJuh78xkn8Ton9d3mUXu4cIYgV64dSBsye	t	2016-02-08 09:52:13.936295	$2a$10$7BZYavHnO7sKSp/QnJ3/nO8dLx80RWdDiWZvoXS28bZVSPWt4..Aa	2016-02-08 15:46:16.625259
3	Anatolii	Zubritckii	a.zubrit@gmail.com	FBRAS	Epigenomics	junior fellow	Epigenomics, gene editing, cancer, histones	poster	Test	This is a test text for a while.	no	no	2016-02-09 08:51:27.445466	2016-02-09 08:54:44.99926	$2a$10$lpmFEzlLXSpUu/U7s73zGOIDstmU8sqJsChqrjiTxWQmnb5zJ5g/K	\N	f	$2a$10$2Y35pm5KL5r5JTR0jwMQWeB6sPYxDMVIo/A7JCVaYv60xWwB22rQC	t	2016-02-09 08:54:44.998294	\N	\N
5	Valentina	Boeva	valentina.boeva@inserm.fr	INSERM, Institut Cochin	Development, reproduction and cancer	Group leader	Epigenetics, Cancer	oral	Computational analysis of epigenetic landscape and chromatin states in cancer	Epigenetic landscape, i.e., positioning of diverse epigenetic marks along the genomic DNA, changes during tumorigenesis. Analysis of histone modifications already demonstrated several important biological phenomena taking place in cancer: formation of de novo enhancers/super-enhancers and long range epigenetic silencing during tumorigenesis. Further analysis of epigenetic changes can shed light into the mechanisms of cancer development and progression, provide prognostic markers for patient stratification and, for sub-groups of patients, suggest efficient treatment affecting epigenetic states. For instance, a wide panel of ‘epigenetic’ drugs (inhibitors of EZH2, HDAC, BRD4 and CDK7) has been shown to be efficient in a number of cancer cell lines and mouse models. ChIP-seq (chromatin immunoprecipitation and sequencing) is the most commonly used technique to characterize DNA binding profiles of DNA-associated proteins: transcription factors, histone modifications, histone variants and polymerase binding. In my presentation, we will discuss how ChIP-seq data can be used in cancer studies to annotate chromatin states and assess changes of the epigenetic landscape. In particular, I will present methods we have developed to normalize profiles and detect regions of enrichment in chromatin marks for ChIP-seq/DNAse-seq/ATAC-seq datasets coming from cancer samples. Our methods take into account copy number aberrations common in cancer genomes, library preparation and sequencing biases and variable signal-to-noise ratio between different experiments. I will also present our recent results on the analysis ChIP-seq profiles of 19 neuroblastoma cell lines, where we characterized the most potent de novo super-enhancers and associated them with neuroblastoma development.	no	no	2016-02-16 09:22:45.305913	2016-02-16 09:24:45.394193	$2a$10$UsZIOtUOQSgcWnPETgVhTOpILE6FEQIwL3/Zh.LwYnNxMU1gW6vgO	\N	f	$2a$10$FgYUNxa5hLA9EEdLus6LpOaj070av6t0T70zHqxZCqj.BAAu06gDO	t	2016-02-16 09:24:27.425774	\N	\N
10	Ivan	Antonov	ivan.antonov@gatech.edu	Research Center of Biotechnology RAS	Laboratory of Genetics and Epigenetics	Postdoctoral research scientist	Long noncoding RNAs, epigenetics, RNA-RNA interactions, RNA-DNA triplexes, ribosomal frameshifting, sequencing errors, gene prediction	oral	Predicting antisense interactions of long noncoding RNAs in human cells	TBA	no	no	2016-02-18 12:40:58.359905	2016-02-18 12:43:18.50091	$2a$10$fJhFOXrOCaSRx1.KYBH/eemBnkw3KZtP7oZtP/ROeBUNM2kwHKyJa	$2a$10$yeKyA4ynWu28GznVFBdDP.3ljeAcNQhlu8cpYgzdf6zoZ2Ge0TrIe	f	$2a$10$Gc1dqlJ0WR8BCZyJjz9y/eWRNlrdv8BcPNhgpWZ5JC4xOo3YNCXpS	t	2016-02-18 12:42:29.412525	\N	\N
6	Finn	Drabløs	finn.drablos@ntnu.no	Norwegian University of Science and Technology	Department of Cancer Research and Molecular Medicine	Professor	Bioinformatics; Gene regulation; Epigenetics	oral	TBA	(will be submitted soon)	yes	yes	2016-02-17 18:21:25.710076	2016-02-17 18:23:04.20668	$2a$10$oBmcHi8ZYyFTJgkfIZ1zO.JHo4t.bRAnT2azzDoCPv.MxDn54gFWy	\N	f	$2a$10$6jJKpKbql/mDk/SbXDwdBu0SZ/iQsIHKIqYVDIoP9Lz3/FZvMx5ce	t	2016-02-17 18:23:04.205129	\N	\N
7	Carsten	Daub	carsten.daub@ki.se	Karolinska Institutet	Biosciences and Nutrition, Science for Life Laboratory 	Associate Professor	bioinformatics, transcriptomics, gene regulation, medical informatics 	oral	[TBD]	[TBD]	no	yes	2016-02-18 07:32:09.012066	2016-02-18 07:33:38.281641	$2a$10$pYVxAY67E1OHz5gPo6geuOCn/jsRtfy8hAh29tnU6UoFsXnBVFmce	\N	f	$2a$10$2XXSdvkvzhlJsw044OMvn.IGWaitvBCa4ZXDgQ8zr8gvssdSR9TXi	t	2016-02-18 07:33:38.27873	\N	\N
8	Yulia	Medvedeva	ju.medvedeva@gmail.com	Research Center of Biotechnology RAS	Laboratory of Epigenetics 	Research Group Leader	Epigenetics	oral	Genetic determinants of epigenetics	TBA	no	no	2016-02-18 10:27:28.969531	2016-02-18 10:28:57.733237	$2a$10$2yoRP1OGkh8NHmkmK2u2UumkqWA7sNQ7dTJl0KjUHtJ6ve106OAaG	\N	f	$2a$10$M21FkyoBVP/lSXdKrZdFq.KdzxqDN27tU4GIo5PepYBLBfClerS6W	t	2016-02-18 10:28:57.73122	\N	\N
11	Anna	Shmelkova	msshmelkova@gmail.com	Skoltech	BioMed	PhD student	Neuroscience, evolution, OMICS data analysis	poster	Attendee without presentation	Attendee without presentation	no	no	2016-02-18 14:39:39.279571	2016-02-18 15:11:06.377593	$2a$10$.adyxdKtxzWcWje57pc0r.FWvUCSE86spbxSJ8f5yRbyB51Ru3y/e	\N	f	$2a$10$9opikCEAaBD42FJ0u8Omme650CpkobzAaiQrmlU44jeYuNbb./1qe	t	2016-02-18 15:11:06.371894	\N	\N
9	Anna	Lioznova	lanna2009@gmail.com	Research Center of Biotechnology RAS	Laboratory of Genomics and Epigenomics of Vertebrates	junior researcher	bioinformatics	poster	Some nice title	An interesting abstract	no	no	2016-02-18 10:44:40.0997	2016-03-01 19:30:27.309385	$2a$10$gPAJRa.8MUVvSGPV1qbNy.qqgQ3rx4S04UABeX4sXb1TkHf/VsWSO	\N	f	$2a$10$Mpj2vBGDRPzJvZ7R0A5TeuWxA3eeYgbZU6lR7ZpFQyyquMX.tdx9C	t	2016-02-18 10:46:33.137706	$2a$10$iBSYcwXzBikF55YI1PNQjufSTJD.NOGfEHJ3I7x5PwWUVXv4HTdHy	2016-03-01 19:27:05.949306
12	Pavel	Baranov	p.baranov@ucc.ie	University College Cork	School of Biochemistry and Cell Biology	Principal Investigator	Translatomics	oral	Why do we know so little about tranlsation?	To be written.	no	yes	2016-02-19 17:13:11.436288	2016-02-19 17:18:46.086687	$2a$10$OYP4bE379hwtvXqyodcZcuS8eCt77hPo6p/8OJKYyDphlRW3.hXnS	\N	f	$2a$10$P/GDpXpFjjoQHEEreXByTeH2UW9d08Zvfki1sTSBP.cx2Ik3nWgY2	t	2016-02-19 17:15:34.429803	\N	\N
13	Eugene	Korotkov	bioinf@yandex.ru	Federal State Institution «Federal Research Centre «Fundamentals of Biotechnology» of the Russian Academy of Sciences»	Group of mathematical methods	Leadering scientist, professor	Developing of mathematical algorithms for dna sequence analysis	oral	New mathematical approach for search of multiple alignments of nucleotide and amino acid sequences	The development and application of mathematical methods in the study of symbolic sequences is of particular importance to achieve great success in the sequencing of various genomes. Latent periodicity is one of the structural regularities of sequences and is widely represented in amino and DNA sequences [1]. A periodicity is considered as latent if the similarity between any two periods is not statistically significant or if it belongs to the twilight zone. Perfect periodicity can become latent periodicity if it accumulates over 1.5 mutations per amino acid in the studied sequence [2]. The distinctive property of latent periodicity is that it cannot be detected by pairwise comparisons of amino acid sequences. However, latent periodicity can be found if we apply a mathematical method to directly detect the multiple alignment of amino acid sequences without constructing pairwise alignments. The periods of a sequence with latent periodicity are sequences for multiple alignment and the multiple alignment can be statistically significant. The goal of this study was to find multiple alignments of amino acid sequences (periods) in the absence of statistically important pairwise alignments.\r\nNew mathematical method for search of a multiple alignment was developed. The method is the new decision of NP-full task for search of a multi alignment. The method uses the direct optimization of the position-weight matrix for multiple sequence alignment without using pairwise alignments by the genetic algorithm. We used the 20n space and random position-weight matrixes where n is the length of period. The developed algorithm was applied to search for latent periodicity with insertions and deletions in the amino acid sequences. This study showed the presence of latent periodicity with insertions and deletions in the amino acid sequences of proteins, for which the presence of latent periodicity was not previously known. The developed method permits to find the multiple alignments of very diverged amino acid and nucleic acid sequences where it is impossible to find multiple alignments by methods developed before. For example, we created the multi alignment of bacterial promoter sequences and very diverged protein families. \r\n\r\n[1]\tKorotkov, Korotkova, Kudryashov, Information decomposition method to analyze symbolical sequences, Phys. Lett. Sect. A Gen. At. Solid State Phys. 312 (2003) 198–210.\r\n[2]\tY.M. Suvorova, M.A. Korotkova, E. V Korotkov, Comparative analysis of periodicity search methods in DNA sequences., Comput. Biol. Chem. 53 Pt A (2014) 43–48.\r\n	no	no	2016-02-27 12:43:09.389721	2016-02-27 12:44:26.280653	$2a$10$BVJw9Rwic3R1o76eKBDxr.oxhoJy.2EU.13y6QIvORiHrk6oAvkIa	\N	f	$2a$10$PvwjP3W08uEXDKAD5Lbebeiw9zS9.Pbl3RCstk09GP0SqxzzSYG0q	t	2016-02-27 12:44:26.278185	\N	\N
14	Irina	Tsvetkova	i.tsvetik@gmail.com	Research Institute of Children's Infectious	Medical Microbiology and Molecular Epidemiology	researcher	Microbiology, Molecular Epidemiology, Phylogenetics	poster	Genetic structure of Streptococcus pneumoniae population	Streptococcus pneumoniae is one of the main pathogens of upper and lower respiratory tract bacterial infections. Pneumococcal diseases prevention is based on immunization with vaccines, which contain from 10 till 23 of the most prevalent capsular polysaccharides from 93 described variants (serotypes). Pneumococcal cohort immunization leads to changes in the population structure of these bacteria. In particular, vaccinal serotypes disappear from circulation and dominant serotypes arise among those strains, which have been considered “rare” before. In some cases, change of the serotype pattern of pneumococcal populations comes out of complex genetic processes, related to horizontal gene transfer.\r\nAnalysis of the pneumococcal populations structure and monitoring of evolutionary processes, which are the consequence of the pneumococcal immunization, are indispensable for the formation of effective prevention strategy. However, currently applicable methods for typing of pneumococci (Serotyping, Multiple Loci Sequence Typing) have insufficient resolution. The most promising method for typing pneumococci is whole-genome sequencing (WGS). Currently, serogroup 19, represented by two serotypes (19A and 19F), is one of the most prevailing serotypes in Russian Federation. Structure analysis of designated serotypes cps-loci, which contains the genes responsible for the synthesis of capsular polysaccharides, have considerable theoretical and practical interest.\r\nObjective:\r\nTo estimate the structure of S.pneumoniae population, represented by serotypes 19F and 19A; to perform the comparative bioinformatic analysis of cps-loci for detection the fact of serotypes switching.\r\nMethods\r\nStrains, received in Scientific Research Institute of Children's Infections\r\nSerotyping was performed by PCR with CDC recommendation.\r\nMultilocus sequence typing (MLST) was performed with standard scheme (www.mlst.net).\r\nWhole-genome sequencing (WGS) was performed on IonTorrent technique (Life Technologies) / SPN2 isolate and MiSeq (Illumina) / SPN133/3733 isolates.\r\nWGS data selection from the published large scale studies and following analysis\r\nWGS data (short reads, HiSeq/Illumina) for selected S.pneumonie population were obtained from the open database of longstanding large scale study (Croucher N.J et al, 2013).1\r\nQuality of the reads and their processing were performed with FASTQC and Trimmomatic softwares. De novo genome assembly was performed with SPAdes v.3.5.0 and SSPACE v.3.0. Reads mapping onto reference genome was performed with Samtools and Bowtie2.\r\nCps loci was annotated using available annotation databases.\r\nResults\r\nThe structure of S.pneumoniae population\r\nS.pneumoniae population (150 isolates), represented by serotypes 19F and 19A, included: 3 isolates, received in Scientific Research Institute of Children's Infections in 2012 from children with acute otitis media and community-acquired pneumonia (2 strains of 19F serotype and one strain of 19A serotype); 41 pneumococci genome sequences from GenBank database (12 strains of 19A serotype and 29 strains of 19F serotype); WGS data (short reads) for 106 isolates from the open database (Croucher N.J et al, 2013), received from the child-carriers in 2001, 2004 and 2007 years (10 strains of 19A serotype and 10 strains of 19F serotype, 18 strains of 19A serotype and 16 strains of 19F serotype, 45 strains of 19A serotype and 7 strains of 19F serotype, respectively). All analyzed population included the pneumococcal isolates, circulated within several years from the time of implementation of 7-valent Prevnar vaccine.\r\nGenomes for 109 strains (missing in GenBank database) were de novo assemled. The assembly pipeline gave on average a total genome length of 2 161 240 bp with average contig length of 33 191 bp and average N50 of 65 656 bp. \r\nTo estimate the whole population structure, reads for 109 strains were mapped onto reference genomes S.pneumoniae: 19F (Taiwan 19F (PMEN14)) or 19A (TCH8431/19A). Mutations frequency for each serotype was estimated. The number of transitions and transversions for the entire population was ranged from 3518 to 14077 and from 136 to 4746, respectively. The average TS/TV for the entire population was corresponded to 2,6, it was evidence of a directional mutations, which had biological significance. To estimate the population clonality, mapped genome sequences were analysed with BAPS v6.0 statistical software.\r\nFor cps-loci comparison, corresponding sequences were extracted from de novo assembled genomes and following estimation of recombinations in variable regions was performed.\r\nConclusion\r\nAnalysis of the genetic structure of the S.pneumoniae populations is important for understanding of epidemiology and disease prevention measures planning.	no	no	2016-03-02 21:40:16.47749	2016-03-02 21:41:45.219916	$2a$10$xj/k154TiDEDFWgrZyXg4.OaUwCtOxQtPwVHveNKmbWd6/cv8nz7i	\N	f	$2a$10$e7xxzoLwHqTTTrrdbuZlPuP8DLS/cUbrt5t93QWhqf4ihbnZEcOV6	t	2016-03-02 21:41:45.21875	\N	\N
15	Yuriy	Orlov	orlov@bionet.nsc.ru	Institute of Cytology and Genetics SB RAS	Lab. of behavior neuroinformatics	Head of lab.	Neuroinformatics, bioinformatics, computer genomics, chromosome contacts, behaviour	oral	Analysis of nucleotide contexts of SNP containing sites	Short mononucleotide repeats can cause errors in DNA reads during sequencing, at least for previous Illumina sequencing technologies. Due to technological breakthrough in the next-generation sequencing (NGS) technologies volume of available genomic data exponentially grows each year, including information on natural variability in mammalian genomes. Fast growth of NGS data demands development of new methods of analysis, processing and pre-processing of raw sequence reads. Applications of sequencing analysis include raw data (sequencing reads) in transcripts and personal genome studies, detection of individual genetic polymorphisms (“1000 genomes” project). The study of genomic context of single nucleotide polymorphisms (SNPs) represented in the dbSNP database (http://www.ncbi.nlm.nih.gov/projects/SNP/) is of greater interest. Association of SNP position in human genome with mononucleotide repeats was shown earlier. We updated context analysis of SNP containing sites using novel software based on text complexity estimates and extended applications of SNP analysis from human “1000 genomes” data to rat and mouse genomes. We’d like discuss problems arising for SNP detection my modern NGS technologies. 	no	yes	2016-03-03 13:07:06.828973	2016-03-03 14:13:08.606925	$2a$10$ETtXaQLtIZcgah/cC5.xuuIE1Nn0SPQdLVRqGrvytmejY86p2C1oS	\N	f	$2a$10$1UstHNiN0N4QF5HTaaJHc./3GJa/jUUf4jFwS.opVI9Ymqdkb7kly	t	2016-03-03 14:13:08.605381	\N	\N
17	Ivan	Kulakovskiy	ivan.kulakovskiy@gmail.com	Engelhardt Institute of Molecular Biology	Laboratory of Computational Methods for Systems Biology	senior research fellow	Regulatory genomics, Transcriptomics, Sequence analysis	oral	HOCOMOCO collection of transcription factor binding sites models: expansion, enhancement and practical applications	In the latest release, HOCOMOCO COmprehensive MOdel COllection\r\n[HOCOMOCO v10, http://hocomoco.autosome.ru] provides binding models\r\nfor 6 hundreds of human and almost 4 hundreds of mouse transcription\r\nfactors.\r\n\r\nThe primary collection provides classic mononucleotide position weight\r\nmatrices (PWMs), which were produced using ChIPMunk motif discovery\r\ntool [http://autosome.ru/ChIPMunk/].\r\n\r\nIn addition to basic mononucleotide position weight matrices (PWMs),\r\nHOCOMOCO includes dinucleotide position weight matrices based on\r\nChIP-Seq data. For motif finding in large sequence sets we provide the\r\ncommand line tool, SPRY-SARUS (SuperAlphabet Representation Utilized\r\nfor motif Search), able to efficiently handle both mono- and\r\ndinucleotide PWMs.\r\n\r\nChIP-Seq data for HOCOMOCO v10 motif discovery and benchmarking was\r\nextracted from GTRD database [http://gtrd.biouml.org]. For all the\r\nmodels we provide quality ratings that are based on a comprehensive\r\nbenchmark study or inherited from HOCOMOCO v9.\r\n\r\nWe present a complete workflow used to construct the last\r\nHOCOMOCO release. In addition, we present results of case\r\nstudies, such as analysis of non-coding somatic mutations in human\r\ncancer, and identification of key transcriptional regulators of mRNAs\r\nthat are translationally regulated by the mTOR signaling pathway.\r\n\r\nThis study was supported by RFBR grants 15-34-20423 and, partly, 14-04-01838.	no	no	2016-03-13 14:58:43.493309	2016-03-17 11:43:07.72278	$2a$10$y3ShkCQm6J3Jfp6V2Q5km.d3ByS6EPGlksoILCkL670rTI4pau6LW	\N	f	$2a$10$Qlh0N.NCyoIHDkKwAVXZcOll.1MIEnqhLamYKmFk6FHTe7ibDaKZC	t	2016-03-13 15:00:45.517532	\N	\N
19	Zoe	Chervontseva	chezoya@gmail.com	Lomonosov Moscow State University	The Faculty of Bioengineering and Bioinformatics	PhD student	bioinformatics, comparative genomics, non-coding RNA, evolution of transcriptional regulation in bacteria	poster	Novel comparative genomic approach for detecting nonhomologous RNA regulatory elements	Precise coordination of ribosomal proteins biosynthesis is crucial for appropriate assembly of ribosomes. More than half of the ribosomal proteins in E.coli are known to be controlled by distinct RNA regulatory elements that occur within their own mRNA. In some cases such regulatory elements mimics the ribosomal RNA site where one or two of regulated proteins may bind providing direct negative feedback from levels of ribosomal proteins. But it seems like this mimicry can be inherent to completely different nonhomologous secondary structures in distinct species. That is why minimal free energy (MFE) based search looks more preferable than methods using evolutionary conservation of RNA secondary structures. In this work we use novel MFE based comparative genomic approach for detecting such nonhomologous RNA regulatory elements and some new predicted ribosomal proteins' RNA-regulators. 	no	no	2016-03-13 17:16:50.807433	2016-03-14 19:44:46.304314	$2a$10$El5gjAgy60c4k/hTpScL3ujbtO0sJkbZanPcsdgK3ggZ0NwKuLDUm	\N	f	$2a$10$nW4gyWd067fffZVQGOXff.lh3d/ijilwDaqkx7vTyysEAryOpKuOi	t	2016-03-13 17:18:15.842742	\N	\N
16	Denis	Vorobyev	denis.g.vorobiev@gmail.com	none	none	none	RNA strutural genomics	poster	Two methods to calculate E-value of RNA of a definite shape	A model of RNA secondary structure is considered which is defined by a given tree topology and a given size range of every i-th element (stem or loop). Additionally, size ranges of the total pattern length or some of its subpatterns can be set.\r\n\r\nIn this work, two methods of calculating the occurrence frequency of such pattern in a random sequence are described. Along with suitably arranged complementary pairs, energy is also taken into account.\r\n\r\nDifferent background models of the random sequence are discussed: from the fixed mononucleotide composition (Bernoulli) model and the fixed dinucleotide composition model to more complex ones. \r\n\r\n	no	no	2016-03-12 17:48:36.304603	2016-03-14 15:15:04.884498	$2a$10$sE2pHrYaodjS3NMqoD/bp.t9hwIOtnva/vNk1qlZOKimzhxOyN4B6	$2a$10$1ogjrp9f15TixIaCY3ldt.g/9gfhLtX6ZNXAogk6Ve3Y0TMCxkZLK	f	$2a$10$E0ZS0P1jbho49Pt9xFa6l.TOHZNRIa2BvmodWtivJmSMKBVJsB0Ku	t	2016-03-12 17:53:17.896064	\N	\N
20	Artem	Kasianov	artem.kasianov@gmail.com	Vavilov institute of general genetics	Laboratory of system biology and computational genetics	Research fellow	de novo assembly algorithms, transcriptomics, metagenomics, computational molecular evolution	oral	Identifying genes of antimicrobial peptides in transcriptomes of Triticum kiharae dorof.et Migush.	Identifying genes of antimicrobial peptides in transcriptomes of Triticum kiharae dorof.et Migush.\r\n\r\nArtem S. Kasianov, Alexey S. Kovtun, Tatyana V. Korostyleva, Ekatherine A. Istomina,  Vsevolod J. Makeev, Alexander M. Kudryavtsev, Tatyana I. Odintsova,  \r\n\r\nContamination of plants with pathogens is usually followed by immune response, that includes synthesis of various antimicrobial compounds and antimicrobial peptides (AMP) are the most important of them. AMPs in plants are short (<100 a.a.), positively charged, cysteine-rich polypeptides that are able to respond to wide spectrum of pathogens. The goal of this work is identification of transcripts that code AMP in healthy and infected by Fusarium oxysporum seedlings of Triticum kiharae dorof.et Migush. wheat using methods of high-throughput sequencing (NGS). The cDNA libraries were sequenced on Illumina HiSeq2000. We developed a method of in silico searching of AMPs in the obtained transcripts. Firstly, we used a method of hidden Markov models, which were created from already known AMPs from A.thaliana and O.sativa. Secondly, we wrote a program in Perl that uses regular expressions for searching AMPs by their cysteine-rich motifs. As a result, we identified more than 500 amino acid sequences of AMPs that correspond to lipid transport proteins, defensins, hevein-like peptides and many others. The lipid transport proteins appeared to be the most widespread family in our resulting set – approximately half of the total number of AMPs. We also identified 145 transcripts of defensins and 3 transcripts coding hevein-like peptides. The majority of identified transcripts are new, previously not described in wheat. Because the genome of T.kiharae consists of subgenomes, which originate from T.urartu and A.tauschii, we showed, which of our identified transcripts refer to these subgenomes and which are individual for T.kiharae. We showed that the expression of some identified genes of AMPs enhances when wheat is infected with the pathogen. Aside from the known families of plant AMPs we identified cysteine-rich peptides with new cysteine motifs. Their role in plants needs a further investigation. Our work is supported by RFBR grants №№ 15-04-0468015/15 and 15-29-02480.\r\n	no	no	2016-03-13 20:22:23.771047	2016-03-13 20:23:47.568994	$2a$10$zd7lxMeVtN6007KHfejTbOBfFPdK7nPqlFnzkhtC1U0p3g9FFIGt6	\N	f	$2a$10$9hOQvSZV69bFC6b6l2kTCeWiwiTqOfLl2sNNYNoFb7lIH6u/Zg8o.	t	2016-03-13 20:23:47.56821	\N	\N
21	Daria	Avdanina	daria.avdanina@biengi.ac.ru	Research Center of Biotechnology RAS	Institute of Bioengineering RAS	scientist	visual system of invertebrates	poster	-	-	no	no	2016-03-14 15:12:07.179319	2016-03-16 12:53:37.747906	$2a$10$l4PC24.zYNWux/vtJr3VEubm2jjv4EXEUWzcRuLrx71bBYYDhykRq	\N	f	$2a$10$oQgBrXnI9k0gP6xoir/gjuIi2.L9oBN49ElLHKDr/sWs.vlNyW4Aa	t	2016-03-16 12:53:37.741604	\N	\N
22	Yaroslav	Popov	syav.popoff@yandex.ru	Moscow State University	Faculty of Bioengineering and Bioinformatics	student	structural biology 	poster	StructAlign - a program for alignment of structures of DNA-protein complexes	Comparative analysis of structures of complexes of homologous proteins with DNA is important in the analysis of DNA–protein recognition. Alignment is a necessary stage of the analysis. An alignment is a matching of amino acid residues and nucleotides of one complex to residues and nucleotides of the other. Currently, there are no programs available for aligning structures of DNA–protein complexes. We present the program StructAlign, which should fill this gap. The program inputs a pair of complexes of DNA double helix with proteins and outputs an alignment of DNA chains corresponding to the best spatial fit of the protein chains.	no	no	2016-03-14 15:43:18.168918	2016-03-14 15:44:40.347136	$2a$10$s/pjqXjzOykVeRGkVKEyLeLSi4K2ESJsouxF6uzB4v4dWXfDwjDDe	\N	f	$2a$10$CJ5rmi5IUHZDV2SsIswoN.VwSeUCri/y41tnIk54TTLcBrSMR8cB6	t	2016-03-14 15:44:40.342578	\N	\N
18	Anna	Obraztsova	alcedooo@gmail.com	Lomonosov Moscow State University	Faculty of bioengineering and bioinformatics	student	Regulation of gene expression, RNA secondary structure prediction, non-coding RNAs	poster	Novel comparative genomic approach for detecting nonhomologous RNA regulatory elements	Precise coordination of ribosomal protein synthesis is crucial for appropriate assembly of ribosomes. More than half of the ribosomal proteins in E.coli are known to be controlled by distinct RNA regulatory elements that occur within their own mRNA. In some cases such regulatory elements mimic the ribosomal RNA site where one or two of regulated proteins may bind providing direct negative feedback from levels of ribosomal proteins. But it seems like this mimicry can be inherent to completely different nonhomologous secondary structures in distinct species. That is why minimal free energy (MFE) based search looks more preferable than methods using evolutionary conservation of RNA secondary structures. In this work we use novel MFE based comparative genomic approach for detecting such nonhomologous RNA regulatory elements and some new predicted ribosomal protein RNA regulators.	no	no	2016-03-13 17:15:12.332893	2016-03-15 08:48:37.730273	$2a$10$bgDhAGkFuzh0ETt3OmUu8unH8UM3.O6DiatNffMP7ZIwdHZ1rZZke	$2a$10$xVbFo.ZVuBA1sfTx3hYNxe.ssXsH.G6tJhz27ZGj6xlElhbp2bfIu	f	$2a$10$9mUL38Lib.R3s9unLelZHO7wZQki3025RqM0Ek5W5ce.mD2Y0kLEG	t	2016-03-13 17:16:39.563802	\N	\N
24	Tatyana	Gorbacheva	gorb.tanya@gmail.com	Voronezh State University	genetics, cytology and bioengineering	professor assistant	molecular genetics, cancer research, DNA sequencing, NGS, NGS data analysis	poster	Free circular DNA as a biomarker of metastasis	.\tThe development of novel techniques for the evaluation of blood biomarkers (liquid biopsies) in cancer  such as circulating tumor cells (CTCs) and circulating tumor DNA (ctDNA) or tumor-specific cell free DNA (cfDNA) have changed the translational medicine as a non-invasive biomarker. Circulating biomarkers might represent both primary tumor and metastatic deposits and provide ways for investigating metastatic processes. The present study investigated the levels of patient-specific mutations in circulating cell-free DNA (cfDNA) in plasma from patients with clear-cell renal cell carcinoma. Pairs of normal-tumor tissue were sequenced, analyzed to establish the mutation profile for each patient and allele-specific PCR were conducted to monitor mutations in cfDNA in plasma. It was observed that there is a tendency of increasing the level of mutant alleles in plasma in patients with metastasis. 	no	yes	2016-03-15 09:40:42.721704	2016-03-15 09:42:13.569878	$2a$10$o2zsiYUWdecIzvZjhXyNmeHgrt7Rlqn2X46uf/dnwKc.lEcAE9nlS	\N	f	$2a$10$.QnoKDTgAaM6xYaBtNZQleHPSlA1T5RMeoZCCvQQc2/FzaFN.Cs5y	t	2016-03-15 09:42:13.567854	\N	\N
23	Anna	Petukhova	annet.057@gmail.com	I.M. Sechenov FMSMU	Faculty of Pharmacy	Student	Neurobiology, bioinformatics, genomics, genetics, epigenetics, molecular biology, oncology, aging. 	oral	Bioinformatic approaches to building of gene regulatory networks.	One of the main tasks of the post-genomic molecular biology and genetics is the researching of principles of organization and functioning of gene regulatory networks - molecular-genetic systems that provide processes of formation of phenotypic features of the organism.\r\nThe appearance of high-performance methods of molecular biology has resulted in the accumulation of a huge amount of experimental data about structural and functional organization of gene networks and their molecular and genetic components. This information is presented in the large number of publications and computer data bases which are describing different aspects of gene networks functioning. Data about gene networks important for medicine and biotechnology is accumulating especially fast and require instant processing and analysis. Analysis of these experimental data is imposible without using of contemporary information technologies and effective mathematical methods of data analysis and modeling of biological systems and processes. As a tool that is indispensable for these needs bioinformatics has a wide range of applications for building of gene regulatory networks and also can make this prosses much easier. This review presents basic bioinformatic approaches to building of gene regulatory networks 	no	no	2016-03-14 22:01:18.913719	2016-03-14 22:04:04.127638	$2a$10$UcYBt.EX45.OoDEctrUN9O6.NNtiroWpaDL7YIVceKu6/lM4DewC2	\N	f	$2a$10$0j9CIk8fQ23yhWEWrI0YOeMGHeFQyx5vG22U3zRESyg4OBd/4yl6G	t	2016-03-14 22:04:04.126635	\N	\N
25	Damir	Terkulov	damir.terkulov@biengi.ac.ru	Research Center of Biotechnology RAS	IT	administrator	...	oral	No title	-	yes	yes	2016-03-16 10:27:55.937221	2016-03-16 10:56:05.614293	$2a$10$hPipL5LVWicvarIX3L7KBu2qgenbuTV2o/95RL0gs8kc4nVsXFD96	\N	f	$2a$10$kbV2fIiMQSAxBni/f4Z.vOWp.2gj6GUGeIZ5FHHSQStDJ/m0jo47m	t	2016-03-16 10:56:05.61144	\N	\N
26	Elena	Osipova	eva-0@mail.ru	Kazan Institute of Biochemistry and Biophysics, Kazan Scientific Center, Russian Academy of Science 	molecular biology	reseacher	protein bioinformatics, NGS	oral	EVOLUTION HISTORY OF LIPOXYGENASE PATHWAY ENZYMES 	The evolution of metabolism of taxonomic distanced organisms is associated with changes of signaling pathways including lipoxygenase pathway. The lipoxygenase specificity defines the site of oxygen incorporation fatty acid during hydroperoxides production, designate the courses of following signaling. In plants the hydroperoxy fatty acid is converted by CYP74 family cytochrome P450 enzymes (allene oxide synthase, hydroperoxide lyase or divinyl ether synthase). In other organisms there are CYP74-like enzymes, but they are not observed in all kingdoms. The way of lipoxygenase signaling pathway evolution is not evident.\r\nLipoxygenase and P450 enzymes nucleotides and amino acids sequences were taken from Uniprot.org and Nelson databases. Biological information was taken from PubMed (NCBI), Brenda-enzymes.org, Uniprot.org, Kegg.jp et al. The multiple alignment were performed by MAFFT, ClustalW and Muscle in MEGA. Phylogeny trees were reconstructed using MAFFT and MEGA. \r\nWe performed the reconstruction of evolution history of lipoxygenase pathway enzymes, which made clear the sequences of lipoxygenases and cytochromes P450 common ancestors. The lipoxygenase specificity variations were provided by duplications of ancestor sequence and subsequent specificity changes. For example, the lipoxygenases of plants are divided into groups according to the specificity catalytic action. The sequence of P450 cytochromes ancestor is analogous to present-day hydroperoxide lyases and allene oxide synthases of CYP74 family of P450. Based on the phylogenetic analysis of biochemically tested lipoxygenases we identified the determinants of substrate specificity (arachidonic or the linolenic acids) of lipoxygenases. It was revealed that the determinants of catalytic specificities of plants lipoxygenases and CYP74 family cytochromes P450 were the unique sites of protein sequences. It was ascertained that catalytic specificity of mammalian lipoxygenases were not depended on the single sites of protein sequences. Moreover, some mammalian lipoxygenases perform the catalytic reactions as plant CYP74 family cytochromes P450 enzymes. Some common properties of lipoxygenase pathway enzymes evolution of organisms from different kingdoms were revealed.	no	no	2016-03-16 13:54:17.029052	2016-03-16 13:56:08.012303	$2a$10$OQuC0/.nPUu2z/2MR6y1L.DoL6GmX76kevQsYb3zWkun8dfI4cFdu	\N	f	$2a$10$TfCej6nPGWIWRFBSXzJOueQ.8KgH5rgQlN6Jx78mDDeBGrveHL63K	t	2016-03-16 13:56:08.010362	\N	\N
\.


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: ielvnfqezwcorg
--

SELECT pg_catalog.setval('"users_id_seq"', 26, true);


--
-- Name: users_pkey; Type: CONSTRAINT; Schema: public; Owner: ielvnfqezwcorg; Tablespace: 
--

ALTER TABLE ONLY "users"
    ADD CONSTRAINT "users_pkey" PRIMARY KEY ("id");


--
-- Name: index_users_on_email; Type: INDEX; Schema: public; Owner: ielvnfqezwcorg; Tablespace: 
--

CREATE UNIQUE INDEX "index_users_on_email" ON "users" USING "btree" ("email");


--
-- Name: unique_schema_migrations; Type: INDEX; Schema: public; Owner: ielvnfqezwcorg; Tablespace: 
--

CREATE UNIQUE INDEX "unique_schema_migrations" ON "schema_migrations" USING "btree" ("version");


--
-- PostgreSQL database dump complete
--

